src/actions/index.test.js

```diff
@@ -0,0 +1,50 @@
+import { NETWORK_KINDS } from '../constants/network';
+import {
+  NEW_NETWORK,
+  LOAD_NETWORK,
+  CHANGE_NETWORK_PROPERTY,
+  ADD_NODE,
+  newNetwork,
+  loadNetwork,
+  changeNetworkProperty,
+  addNode,
+} from './index';
+
+describe('index file tests', () => {
+  it('newNetwork ', () => {
+    expect(newNetwork).toBeInstanceOf(Function);
```

Comment:
I believe you don't need this test: checking if is a `Function` for this and all other methods.
If it changes to be not a `Function` (what it probably won happen) the other tests about this action will break 😄